examples/example_graph.jsonld

```diff
@@ -0,0 +1,53 @@
+{
+    "@context": {
+        "uco-tool": "https://ontology.unifiedcyberontology.org/uco/tool/",
+        "uco-observable": "https://ontology.unifiedcyberontology.org/uco/observable/",
+        "uco-core": "https://ontology.unifiedcyberontology.org/uco/core/",
+        "case-investigation": "https://ontology.caseontology.org/case/investigation/",
+        "uco-identity": "https://ontology.unifiedcyberontology.org/uco/identity/",
+        "uco-location": "https://ontology.unifiedcyberontology.org/uco/location/",
+        "kb": "http://example.org/kb/",
+        "uco-vocabulary": "https://ontology.unifiedcyberontology.org/uco/vocabulary/",
+        "uco-types": "https://ontology.unifiedcyberontology.org/uco/types/",
+        "rdfs": "http://www.w3.org/2000/01/rdf-schema#",
+        "rdf": "http://www.w3.org/1999/02/22-rdf-syntax-ns#",
+        "xsd": "http://www.w3.org/2001/XMLSchema#",
+        "uco-action": "https://ontology.unifiedcyberontology.org/uco/action/"
+    },
+    "@graph": [
+        {
+            "uco-core:description": "MSAB is a global leader in digital forensic technology for mobile device examination and analysis. https://www.msab.com/about-us/",
```

Comment:
I request CASE not include any phrasing that sounds like an endorsement of an organization.